testdata/scripts/format.txt

```diff
@@ -5,6 +5,7 @@ stderr 'no Gunk packages to format'
 # formatting a normal package and checking the result
 gunk format .
 cmp echo.gunk echo.gunk.golden
+cmp notutil.gunk notutil.gunk.golden
```

Comment:
you just compare the files? I think you're missing the actual `gunk format` call with files.

also, separate the test from the "formatting a normal package" via an empty line and another section comment.

Reply:
danke, sir